semantic_model_generator/data_processing/cte_utils.py

```diff
@@ -0,0 +1,227 @@
+from typing import List
```

Comment:
please add tests for this file (or, ideally, break into a new PR)

Reply:
those utils are needed for the streamlit code to work. So put it here for now